package.json

```diff
@@ -12,9 +12,9 @@
   },
   "engines": {
     "node": ">=12.10.0",
-    "npm": ">=5.6.0",
-    "yarn": "Yarn is disabled for this project, use npm instead"
+    "npm": ">=5.6.0"
   },
+  "babel": {},
```

Comment:
At least in the version of Babel this project uses, this was interfering with downstream configs; since it doesn't define a config for the package itself, it tries to go up to configs for the parent project, which will fail since you're using an old version of babel.